flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "only-arches": [
+        "x86_64"
```

Comment:
You can build from source for aarch64 support, and easily set up a workflow to automatically update the sources as appropriate.